app/ante/ante.go

```diff
@@ -177,9 +180,37 @@ func IsSystemTx(tx sdk.Tx, isAuthorizedSigner func(string) bool) bool {
 		*observertypes.MsgAddBlameVote:
 		signers := innerMsg.GetSigners()
 		if len(signers) == 1 {
-			return isAuthorizedSigner(signers[0].String())
+			if isObserverAuthorized(signers[0].String()) {
+				return true
+			}
 		}
 	}
 
 	return false
 }
+
+// IsAuthorityTx checks if there is a MsgCreateValidator in the transaction and returns the signer if present.
+func IsAuthorityTx(ctx sdk.Context, tx sdk.Tx, options HandlerOptions) (bool, bool) {
+	for _, msg := range tx.GetMsgs() {
+		if validatorMsg, ok := msg.(*stakingtypes.MsgCreateValidator); ok {
+			signers := validatorMsg.GetSigners()
+			if len(signers) >= 1 {
+				return true, options.AuthorityKeeper.IsAuthorized(ctx, signers[0].String(), authoritytypes.PolicyType_groupAdmin)
```

Comment:
please update readability
```suggestion
				isAuthorized := options.AuthorityKeeper.IsAuthorized(ctx, signers[0].String(), authoritytypes.PolicyType_groupAdmin)
				return true, isAuthorized
```